includes/utils/class-amp-image-list.php

```diff
@@ -0,0 +1,138 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+
+// phpcs:disable Generic.Files.OneObjectStructurePerFile.MultipleFound
+/**
+ * Image collection classes.
+ *
+ * @package AMP
+ */
+
+/**
+ * Interface Has_Caption
+ *
+ * @since 1.5.0
+ */
+interface Has_Caption {
```

Comment:
This either needs prefixing or (better) the `AMP` namespace.

Reply:
Sure, this now [has the AMP namespace](https://github.com/ampproject/amp-wp/pull/3659/files#diff-97f7f9bbd8f82d4b8782110a007991cbR8).